zerver/lib/test_classes.py

```diff
@@ -430,6 +430,15 @@ def get_messages(self, anchor: int=1, num_before: int=100, num_after: int=100,
         data = result.json()
         return data['messages']
 
+    def get_messages_response(self, anchor: int=1, num_before: int=100, num_after: int=100,
+                              use_first_unread_anchor: bool=False) -> Dict[str, List[Dict[str, Any]]]:
+        post_params = {"anchor": anchor, "num_before": num_before,
+                       "num_after": num_after,
+                       "use_first_unread_anchor": ujson.dumps(use_first_unread_anchor)}
+        result = self.client_get("/json/messages", dict(post_params))
+        data = result.json()
+        return data
```

Comment:
Let's not duplicate the code from `get_messages`; we should just have `get_messages` be a wrapper for this that just returns the `messages` part.